ui/src/components/input/QInput.js

```diff
@@ -352,6 +352,14 @@ export default Vue.extend({
     this.autogrow === true && this.__adjustHeight()
   },
 
+  activated () {
+    this.__wasDeactivated !== void 0 && this.autofocus === true && this.$nextTick(() => { this.focus() })
+  },
+
+  deactivated () {
```

Comment:
This is not needed - activated is only triggered if it was previously deactivated

Reply:
Hmm .. as last checked .. Both lifecyclehooks (mounted and activated) in the first run ...

Reply:
Oh, shit...
You are right, it's called right after mounted if it's inside a keepAlive

Reply:
yupp